install/Install.ps1

```diff
@@ -0,0 +1,66 @@
+# PowerShell Script to Install Spice CLI on Windows
+
+$spiceBin = ".spice\bin"
+$spiceCliInstallDir = Join-Path $HOME $spiceBin
+$spiceRepoName = "spiceai"
+$spiceOrgName = "spiceai"
+$spiceCliFileName = "spice.exe"
+$spiceCliFullPath= Join-Path $spiceCliInstallDir $spiceCliFileName
+
+# Ensure the installation directory and auth file exist
+New-Item -Path $spiceCliInstallDir -ItemType Directory -Force
+
+function Get-LatestRelease {
+    $url = "https://api.github.com/repos/$spiceOrgName/$spiceRepoName/releases/latest"
+    $releaseInfo = Invoke-RestMethod -Uri $url
+    return $releaseInfo.tag_name
+}
+
+# Function to download and install Spice CLIv
+function Download-And-Install-Spice {
+    Write-Host "Checking the latest Spice version..."
+
+    $latestReleaseTag = Get-LatestRelease
+
+    Write-Host "Installing Spice $latestReleaseTag"
+
+    $artifactName = "${spiceCliFileName}"
+    $downloadUrl = "https://github.com/$spiceOrgName/$spiceRepoName/releases/download/$latestReleaseTag/$artifactName"
+    $tempPath = [System.IO.Path]::GetTempPath()
+    $tempFile = Join-Path $tempPath $artifactName
+
+    Write-Host "Downloading Spice CLI from $downloadUrl..."
+    Invoke-WebRequest -Uri $downloadUrl -OutFile $tempFile
+
+    Move-Item -Path (Join-Path $tempPath $spiceCliFilename) -Destination $spiceCliFullPath -Force
+
+    # Temporary download Spice runtime as `spice upgrade` is not currently working on Windows.
```

Comment:
I'm not sure I understand the comment in the context of the code below.  What do you mean by this comment?

Reply:
We should not install Spice runtime (spiced) manually as Spice CLI includes special functionality to detect missing or outdated runtime version and download it. This is not currently working on Windows so I'm manually installing runtime as well.  

MacOS/Linux installation script installs CLI only, runtime is downloaded by CLI

Reply:
Updated comment to just `# Download the Spice runtime`